altair/examples/line_chart_with_cumsum.py

```diff
@@ -0,0 +1,23 @@
+"""
+Line Chart with Cumulative Sum
+------------------------------
+This chart creates a simple line chart from the cumulative sum of a fields.
+"""
+# category: line charts
+import altair as alt
+from vega_datasets import data
+
+source = data.wheat()
+
+alt.Chart(source).mark_line().transform_window(
+    # Sort the data chronologically
+    sort=[{'field': 'year'}],
+    # Include all previous records before the current record and none after
+    frame=[None, 0],
```

Comment:
Maybe note that this is the default value, so specifying ``[None, 0]`` here is optional

Reply:
Done